src/Umbraco.Infrastructure/Migrations/Custom/IKeyValueServiceInitialization.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Umbraco.Core.Persistence;
+
+namespace Umbraco.Infrastructure.Migrations.Custom
+{
+    public interface IKeyValueServiceInitialization
```

Comment:
We actually don't need this (or the implementation) at all, this was a hack/workaround for v7 -> v8 migrations

Reply:
Have removed this and all uses of it.